Stylesheets/common_css/general.css

```diff
@@ -0,0 +1,26 @@
+.container {
+    margin: 3rem auto;
+    min-width: 15rem;
+    max-width: 25rem;
+    min-height: 5rem;
+    max-height: 10rem;
+    background-color: var(--st-background-grey);
+    border: 2px solid grey;
+}
+
+.padding-between {
+    padding: 0.5rem 0.5rem;
```

Comment:
use padding: 0.5rem since top-bottom and left-right values are the same

Reply:
thanks, I have changed it